apm-sniffer/apm-toolkit-activation/apm-toolkit-trace-activation/src/test/java/org/apache/skywalking/apm/toolkit/activation/trace/TagAnnotationTest.java

```diff
@@ -296,17 +301,29 @@ public User testMethodWithTags(String username, Integer age) {
             return new User(username, age);
         }
 
-        @Tags({@Tag(key = "username", value = "arg[0]"), @Tag(key = "info", value = "returnedObj.0.info")})
+        @Tags({
+            @Tag(key = "username", value = "arg[0]"),
+            @Tag(key = "info", value = "returnedObj.0.info"),
+            @Tag(key = "info2", value = "returnedObj.[0].info"),
+        })
         public List<User> testMethodWithReturnList(String username, Integer age) {
             return Arrays.asList(new User(username, age));
         }
 
-        @Tags({@Tag(key = "username", value = "arg[0]"), @Tag(key = "info", value = "returnedObj.0.info")})
+        @Tags({
+            @Tag(key = "username", value = "arg[0]"),
+            @Tag(key = "info", value = "returnedObj.0.info"),
+            @Tag(key = "info2", value = "returnedObj.[0].info"),
```

Comment:
I think there should require expected file update too.

Reply:
OK, which e2e test need to be updated?

Reply:
e2e test updated